FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentContentsFinder.cs

```diff
@@ -40,6 +41,19 @@ public unsafe partial struct AgentContentsFinder {
 
     [MemberFunction("E9 ?? ?? ?? ?? 8B 93 ?? ?? ?? ?? 48 83 C4 20")]
     public partial void OpenRouletteDuty(byte roulette, bool hideIfShown = false);
+
+    /// <summary>
+    /// Unchecks all selected duties, this is the same command that "Clear Selection" sends.
+    /// </summary>
+    public void ClearSelectedDuties() {
+        var returnValue = stackalloc AtkValue[1];
+        var command = stackalloc AtkValue[2];
+        // "E8 ?? ?? ?? ?? 0F B6 8E ?? ?? ?? ?? 3B C8"
```

Comment:
Sorry, suggested the wrong sig. This is not where it lands.
Using a signature of the caller instead (inside AddonContentsFinder.ReceiveEvent).
```suggestion
        // "C7 45 ?? ?? ?? ?? ?? BF 03 00 00 00 89 45"
```